oauth-server/src/main/java/io/strimzi/kafka/oauth/server/OAuthKafkaPrincipal.java

```diff
@@ -45,7 +45,7 @@ public static OAuthKafkaPrincipal takeFromThreadContext() {
 
     public static void setToThreadContext(OAuthKafkaPrincipal principal) {
         if (currentTL.get() != null) {
-            throw new RuntimeException("Internal error - current principal already set");
+            throw new RuntimeException("Internal error - current principal already set. Make sure you have 'principal.builder.class=io.strimzi.kafka.oauth.server.OAuthKafkaPrincipalBuilder' in your server.properties");
```

Comment:
The config file is not _necessarily_ called `server.properties`, so maybe call it "...in your broker configuration file".

Reply:
We can do that.